src/renderer/components/ParticipantsTile/AddParticipant.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <form class="form" @submit.prevent="handleSubmit">
+    <b-field>
+      <b-input
+        placeholder="Type name"
+        type="text"
+        v-model="name"
+      />
+      <p class="control">
+        <b-input type="submit" value="Add" />
+      </p>
+    </b-field>
+  </form>
+</template>
+
+<script>
+export default {
+  props: {
+    onAdded: { type: Function },
+  },
+  data() {
+    return {
+      name: '',
+    };
+  },
+  methods: {
+    handleSubmit() {
+      if (this.name) {
+        this.onAdded(this.name);
```

Comment:
This should be changed to `this.$emit('on-added', this.name);`

Reply:
It's how Vue was written, it's not bad on it's own, but the general rule of thumb is that you pass props down and events up. Plus it's more readable - if you see props, you expect (in Vue world) this props to be a props, not a reference to a function.